### ISSUE TITLE
More information on Errors

### ISSUE THREAD

**Issue by bugzpodder** [r-wasm/webr#284]
Not sure if it's doable but it would be great to have a bit more stack trace or even the line number from error messages thrown by evalR

**Comment by georgestagg** [comment]
Agreed. I recognise that the error output is sometimes extremely bare, particularly when using `evalR()` functions directly rather than the R console.
